osu.Game/Overlays/Dashboard/Dashboard/DashboardBeatmapPanel.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.Drawables;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Overlays.Dashboard.Dashboard
```

Comment:
pretty bad namespace choice (why does it have `Dashboard` twice?)

would suggest using something more descriptive (`Dashboard.Components`? `Dashboard.Panels`?)

Reply:
Made this because we have dashboard overlay and dashboard tab in it

Reply:
it reads pretty wrong to me.

on another note, how about `Dashboard.Home`? (as that's where the `home` link leads to, the URL web-side is also `/home`?)